src/HomeWork5/HomeWork5/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace HomeWork5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        { 
+            var cages = new IAnimal[3];
+            cages[0] = new Cow();
+            cages[1] = new Cat();
+            cages[2] = new Dog();
+
+            Console.WriteLine("Выбирете клетку: ");
+
+            var cage = int.Parse(Console.ReadLine());
```

Comment:
TryParse is always a better option to avoid exceptions in case of invalid input, whereas handling exception introduces additional overhead for the runtime